internal/storage/v1/elasticsearch/spanstore/from_domain_test.go

```diff
@@ -48,7 +48,7 @@ func loadFixtures(t *testing.T, i int) (inStr []byte, outStr []byte) {
 	in := fmt.Sprintf("fixtures/domain_%02d.json", i)
 	inStr, err = os.ReadFile(in)
 	require.NoError(t, err)
-	out := fmt.Sprintf("fixtures/es_%02d.json", i)
+	out := fmt.Sprintf("../../../elasticsearch/dbmodel/fixtures/es_%02d.json", i)
```

Comment:
I recommend making a copy into v2/../fixtures

Reply:
in other words, I don't think a shared file `internal/storage/elasticsearch/dbmodel/fixtures/es_01.json` makes sense, each test should have its own fixtures.